### PR TITLE
pin jjb to <5

### PR DESCRIPTION
**Pull request by evgeni**
Fixes: 440f7de76be6fafb553f908e2aef2e9b72a30a0d

### CODE REVIEW

File: theforeman.org/pipelines/infra/updateJobs.groovy [hunk 1]
```diff
@@ -14,7 +14,7 @@ pipeline {
                 ])
 
                 virtEnv('jjb-venv', 'pip install pbr')
-                virtEnv('jjb-venv', 'pip install jenkins-job-builder')
+                virtEnv('jjb-venv', 'pip install jenkins-job-builder<5')
```
Comment (evgeni) [inline theforeman.org/pipelines/infra/updateJobs.groovy:17]: you could argue, this should be a `-r requirements.txt` instead?

**Review by ekohl — APPROVED** [review]

File: theforeman.org/pipelines/infra/updateJobs.groovy (hunk 1 above)
Comment (ekohl) [reply theforeman.org/pipelines/infra/updateJobs.groovy:17]: Indeed. I'd prefer that.

File: theforeman.org/pipelines/infra/updateJobs.groovy [hunk 2]
```diff
@@ -14,7 +14,7 @@ pipeline {
                 ])
 
                 virtEnv('jjb-venv', 'pip install pbr')
```
Comment (ekohl) [inline theforeman.org/pipelines/infra/updateJobs.groovy:16]: Is this still needed? I imagine pip should be able to do this.

Comment (evgeni) [reply theforeman.org/pipelines/infra/updateJobs.groovy:16]: you would, right? (it didn't last time we tried)

**Pull request merged**